### PR TITLE
Republish @VictorSanh's blogpost

### PR DESCRIPTION
**Pull request by julien-c**

### CODE REVIEW

File: simple-considerations.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+*   The learning rate is taking funky values like 0 all the time
+*   Your inputs are being truncated in a suboptimal way
+
+> Pro-tip: when you work with language, have a serious **look at the outputs of the tokenizers**. I can’t count the number of lost hours I spent trying to reproduce results (and sometimes my own old results) because something went wrong with the tokenization.🤦‍♂️
+
+Another useful tool is **deep-diving into the training dynamic** and plot (in Tensorboard for instance) the evolution of multiple scalars through training. At the bare minimum, you should look at the dynamic of your loss(es), the parameters, and their gradients.
+
+As the loss decreases, you also want to look at the model’s predictions: either by evaluating on your development set or, my personal favorite, **print a couple of model outputs**. For instance, if you are training a machine translation model, it is quite satisfying to see the generations become more and more convincing through the training. You want to be more specifically careful about overfitting: your training loss continues to decreases while your evaluation loss is aiming at the stars.💫
+
+4\. 👀 Tune but don’t tune blindly
```
Comment (VictorSanh) [inline simple-considerations.md:102]: nitpicking: Sections 1,2,3, and 4 don't have the same format

**Review by VictorSanh — CHANGES_REQUESTED** [review]
👍 great thanks
expect the format nit, lgtm!

File: simple-considerations.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+<div class="blog-metadata">
+    <small>Published February 4, 2021 – <span class="">Initially published on Medium, Sept 2020.</span></small>
+    <a target="_blank" class="btn no-underline text-sm mb-5 font-sans" href="https://github.com/huggingface/blog/blob/master/simple-considerations.md">
+        Update on GitHub
+    </a>
+</div>
+
+<div class="author-card">
+    <a href="/VictorSanh">
+        <img class="avatar avatar-user" src="https://aeiljuispo.cloudimg.io/v7/https://s3.amazonaws.com/moonup/production/uploads/1590600248871-noauth.jpeg?w=200&h=200&f=face" title="Gravatar">
```
Comment (stefan-it) [inline simple-considerations.md:21]: Oh, Victor has a new haircut 🤔

Comment (julien-c) [reply simple-considerations.md:21]: i'm hardcoding his avatar from huggingface.co/victorsanh, @VictorSanh lmk if you want me to change it

**Review by VictorSanh — APPROVED** [review]

**Pull request merged**